### DISCUSSION TITLE
Cannot download videos

### DISCUSSION THREAD

**Discussion by hectorhalpizar** [aclap-dev/video-downloadhelper#503] [Help and Q&A]
### Your question

Hi. I'm not able to download video. I copied the information.

Thanks.

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "microsoft",
  "channel": "stable",
  "page_url": "undef",
  "error": {
    "details": "SideDownload error: : [mov,mp4,m4a,3gp,3g2,mj2 @ 000001fe444a7ac0] trun track id unknown, no tfhd was found\r\n[mov,mp4,m4a,3gp,3g2,mj2 @ 000001fe444a7ac0] error reading header\r\n[in#0 @ 000001fe444a7900] Error opening input: Invalid data found when processing input\r\nError opening input file https://scontent-lax3-2.xx.fbcdn.net/v/t42.1790-2/162430033_311047737125634_7533907345581570256_n.mp4?_nc_cat=1&ccb=1-7&_nc_sid=9a5d50&efg=eyJ2ZW5jb2RlX3RhZyI6ImRhc2hfYXVkaW9fYWFjcF82NF9mcmFnXzJfYXVkaW8iLCJ2aWRlb19pZCI6MTgzOTAwMzAzMzE0MzU0fQ%3D%3D&_nc_eui2=AeHLjnlBPrQKkwTxjDjE7JZj9ME05s-U20H0wTTmz5TbQdfmCBF0WFGN-gyuSDQngozXVBD0fypq0lH9RckSvHYV&_nc_ohc=2RzceZqAQL4Q7kNvgHLZeac&_nc_ht=scontent-lax3-2.xx&oh=00_AYAOn452oPhwqs7poobHUND6gqndC-C5xF0rS8OvXPTsaw&oe=664D5249&bytestart=116929&byteend=149146.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_6459737361346719",
    "error": "coapp_failure",
    "id": "coapp_failure_514364fd-ba66-4e6c-b31d-293ffb19f38c"
  }
}

**Comment by jcfrog** [comment]
Hi,
Any example?

**Comment by xXxAZRAELxx** [comment]
"Try downloading at Vumoo or about a hundred other sites. The program is great when you find a site it works at. I'd say 7 out of 10 sites this program is a bust. I for one would donate something if the odds of it working would hit at least 50/50!!"

-- Edited by moderator: don't use all caps: https://github.com/aclap-dev/video-downloadhelper/wiki/Code-Of-Conduct

**Reply by paulrouget** [reply]
You need to tell us what website doesn't work.

**Comment by hectorhalpizar** [comment]
This was happening on any website. This happened like almost two weeks ago, but it seems to be working now just fine.

Thanks.
